nav2_amcl/src/pf/pf_vector.c

```diff
@@ -35,7 +35,7 @@
 
 
 // Return a zero vector
-pf_vector_t pf_vector_zero()
+pf_vector_t pf_vector_zero(void)
```

Comment:
We have a number of void functions throughout the stack; why pick out these 2?

Reply:
These were the cases reported as errors by the compiler (compiling as C where the void is required rather than C++ where it isn't).